controller/contactusController.js

```diff
@@ -1,7 +1,28 @@
 // dbClient = require('../dbConnection.js');
 
-const contactus = {
-
-};
-
-module.exports = contactus;
\ No newline at end of file
+const contactus = async (req, res) => {
+    try {
+      const { name, email, message } = req.body;
+      if (!name) {
+        return res.status(400).send('Name is required');
+      }
+    
+      if (!email) {
+        return res.status(400).send('Email is required');
+      }
+  
+      if (!message) {
+        return res.status(400).send('Message is required');
+      }
+      console.log('Received data:', req.body);
+      
+      //TODO Connect to DB to add contact records
+  
+      res.json({ message: 'Data received successfully!' });
```

Comment:
I know the actual saving to the db isnt possible right now but have this return the object that will be created.
This should also probably return a 201 response rather than a 200.

Reply:
resolved